frontend/src/components/regions/CardDetailsModal/Presentation.js

```diff
@@ -62,66 +64,71 @@ const CardBasicDetails = ({ card }) => {
     card.reviewRequestTime && new Date(card.reviewRequestTime).toLocaleString();
   const completeTime =
     card.completeTime && new Date(card.completeTime).toLocaleString();
-
+  
   return (
     <React.Fragment>
-      <Grid container>
-        <Grid item xs={12} sm={12} md={12}>
-          <Typography variant="h5">
-            {card.contentTypeNice}: {card.title}
-          </Typography>
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <TagChips tagNames={card.tagNames} />
-          <FlavourChips flavourNames={card.flavourNames} />
-          <StoryPoints storyPoints={card.storyPoints} />
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Typography variant="subtitle2">Assignees:</Typography>
-            <UsersTable userNames={card.assigneeNames} userIds={card.assignees} />
-            <Typography variant="subtitle2">Reviewers:</Typography>
-            <UsersTable userNames={card.reviewerNames} userIds={card.reviewers} />
-          </Paper>
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Table>
-              <TableBody>
-                <TableRow>
-                  <TableCell>Due Time</TableCell>
-                  <TableCell>{dueTime}</TableCell>
-                </TableRow>
-                {card.startTime && (
-                  <TableRow>
-                    <TableCell>Start Time</TableCell>
-                    <TableCell>{startTime}</TableCell>
-                  </TableRow>
-                )}
-
-                {card.reviewRequestTime && (
-                  <TableRow>
-                    <TableCell>Review Request Time</TableCell>
-                    <TableCell>{reviewRequestTime}</TableCell>
-                  </TableRow>
-                )}
-                {card.completeTime && (
-                  <TableRow>
-                    <TableCell>Complete Time </TableCell>
-                    <TableCell>{completeTime}</TableCell>
-                  </TableRow>
-                )}
-              </TableBody>
-            </Table>
-          </Paper>
-        </Grid>  
```

Comment:
Why are you removing all the grid stuff? Surely we are just trying to add a button. I dont think you should be rewriting the whole file